ArduCopter/Copter.cpp

```diff
@@ -147,6 +147,9 @@ const AP_Scheduler::Task Copter::scheduler_tasks[] = {
 
     SCHED_TASK(rc_loop,              250,    130,  3),
     SCHED_TASK(throttle_loop,         50,     75,  6),
+#if AP_FENCE_ENABLED
+    SCHED_TASK(fence_check,           25,    100,  7),
```

Comment:
might be worth moving fence checking to an IO callback, or add a FENCE_OPTIONS bit for fast fence checking